helm/ph-ee-engine/connector-airtel/templates/deployment.yaml

```diff
@@ -0,0 +1,96 @@
+{{- if .Values.enabled -}}
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ph-ee-connector-airtel
+  labels:
+    app: ph-ee-connector-airtel
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: ph-ee-connector-airtel
+  template:
+    metadata:
+      labels:
+        app: ph-ee-connector-airtel
+      annotations:
+  {{- if .Values.deployment.annotations }}
+  {{ toYaml .Values.deployment.annotations | indent 8 }}
+  {{- end }}
+spec:
+  initContainers:
+    #During this Pod's initialization, check that zeebe-gateway service is up and running before starting this pod
+    - name: check-zeebe-gateway-ready
+      image: busybox:latest
+      command: [ 'sh', '-c','until nc -vz {{ .Release.Name }}-zeebe-gateway 26500; do echo "Waiting for zeebe-gateway service"; sleep 2; done;' ]
+  containers:
+    - name: ph-ee-connector-airtel
+      image: "{{ .Values.image }}"
+      ports:
+        - containerPort: 5000
+      imagePullPolicy: "{{ .Values.imagePullPolicy }}"
+      livenessProbe:
+        httpGet:
+          path: /actuator/health/liveness
+          port: 8080
+        initialDelaySeconds: {{.Values.livenessProbe.initialDelaySeconds}}
+        periodSeconds: {{.Values.livenessProbe.periodSeconds}}
+      readinessProbe:
+        httpGet:
+          path: /actuator/health/readiness
+          port: 8080
+        initialDelaySeconds: {{.Values.readinessProbe.initialDelaySeconds}}
+        periodSeconds: {{.Values.readinessProbe.periodSeconds}}
+      resources:
+        limits:
+          memory: "{{ .Values.limits.memory }}"
+          cpu: "{{ .Values.limits.cpu }}"
+        requests:
+          memory: "{{ .Values.requests.memory }}"
+          cpu: "{{ .Values.requests.cpu }}"
+      env:
+        - name: "springProfileActive"
+          value: "{{ .Values.global.springProfileActive }}"
+        - name: "ZEEBE_BROKER_CONTACTPOINT"
+          value: "{{ .Release.Name }}-zeebe-gateway:26500"
+        - name: "ZEEBE_INIT-TRANSFER_WAIT-TIMER"
+          value: "{{ .Values.zeebeInitTransferWaitTimer }}"
+        - name: "AIRTEL_LOCAL_HOST"
```

Comment:
You can remove this part as airtel doesn’t have callbacks.

Reply:
Done.